packages/design-system/src/styles/components/_Button.scss

```diff
@@ -51,13 +50,11 @@
     color: $color-primary-darkest;
   }
 
-  @if $ds-include-focus-styles {
-    // Make sure active overrides focus styles
-    &:active:focus,
-    &.ds-c-button--active:focus {
-      background-color: $color-white;
-      box-shadow: none;
-    }
+  // Make sure active overrides focus styles
+  &:active:focus,
+  &.ds-c-button--active:focus {
+    background-color: $color-white;
+    box-shadow: none;
   }
```

Comment:
Do we intentionally not show the focus ring in the active state?

Reply:
Yep that's correct 👍  I'm sure there is a closed PR somewhere explaining why but ATM I can't remember the reasoning